presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoIntervalDayTime.java

```diff
@@ -56,6 +57,12 @@ public boolean equals(Object obj)
         return Objects.equals(this.milliSeconds, other.milliSeconds);
```

Comment:
not related to this patch, but why do we box the `milliSeconds` here? We can use the same method that you use in `compareTo` below. same for `PrestoIntervalYearMonth` below.